Src/FluentAssertions/Equivalency/MemberFactory.cs

```diff
@@ -19,5 +20,17 @@ public static IMember Create(MemberInfo memberInfo, INode parent)
 
             throw new NotSupportedException($"Don't know how to deal with a {memberInfo.MemberType}");
         }
+
+        internal static IMember Find(object target, string memberName, Type preferredMemberType, INode parent)
+        {
+            PropertyInfo property = target.GetType().FindProperty(memberName, preferredMemberType);
```

Comment:
Note, I had to double check the implementation to verify that `FindProperty` includes inherited properties.
Have you double-checked that we can map between inherited properties?

Reply:
Another good catch. I've added an additional spec for that.